components/eam/src/physics/cam/cldfrc2m.F90

```diff
@@ -953,7 +962,8 @@ subroutine aist_vector(qv_in, T_in, p_in, qi_in, ni_in, landfrac_in, snowh_in, a
      Kc = 75._r8
    ! Wilson & Ballard closure ( Option.4. smaller = more ice clouds)
    ! Slingo modified (option 5)
-     minice = 1.e-12_r8
+!ASD     minice = 1.e-12_r8 - Should be a namelist option now
+     if (iceopt.eq.7) minice = 1.e-5_r8
```

Comment:
Seems like we should specify minice in the compset defaults for our case rather than having it be a namelist option, but overriding the namelist if iceopt=7...